# Define msg for rail sensors
# int8[2] rail_sensor
int8 rail_sensor